docs/CONTRIBUTING.md

```diff
@@ -25,15 +25,15 @@ The following table lists the ticket type labels we use when there is work to be
 |Ticket type label                  |Description                                                                                                                                                                         |
 |:----------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 |L0 - security                      |A security vulnerability within the Marked library is discovered.                                                                                                                   |
-|L1 - broken                        |Valid usage results in incorrect output compared to [supported specifications](README.md#specifications) OR causes marked to crash AND there is no known workaround for the issue. |
+|L1 - broken                        |Valid usage results in incorrect output compared to [supported specifications](#README.md#specifications) OR causes marked to crash AND there is no known workaround for the issue. |
```

Comment:
nit: I think the urls to md files should being with a slash `#/README.md#specifications`.

Reply:
We determine whether it is a route or anchor by the existence of `/.md$/` but we could change it to `/^\\/`

Reply:
I also realized, we can use `#/README` and drop the `.md` altogether which will read a little bit better.